src/game/server/gamemodes/dbl.cpp

```diff
@@ -37,6 +37,8 @@ int CGameControllerDBL::OnCharacterDeath(class CCharacter *pVictim, class CPlaye
 {
 	IGameController::OnCharacterDeath(pVictim, pKiller, Weapon);
 
+	if (pVictim->GetPlayer() != pKiller)
+	pKiller->AddVictim(pVictim->GetPlayer());
```

Comment:
This does not appear as though it is tabbed over in this view, but in the codebase, it does appear to be tabbed over to indicate that the if statement controls this without the use of curly braces.